ilc/client.js

```diff
@@ -1,7 +1,6 @@
 import * as singleSpa from 'single-spa';
-import deepmerge from 'deepmerge';
 
-import * as Router from './common/router/Router';
+import Router from './common/router/ClientRouter';
```

Comment:
As I understood, we use it only on the client-side
Please move it into the './client'

Reply:
we have router folder, so it's incorrect to put client router to separate folder